docs/pages/distribution/publishing-websites.mdx

```diff
@@ -365,32 +365,22 @@ For example, if the GitHub repository is `expo-gh-pages`, the following will be
 
 <Step label="4">
 
-Because Expo uses underscores in generated files, you need to disable jekyll by creating a **.nojekyll** file in the **public** directory.
-
-<Terminal cmd={['$ mkdir public && touch public/.nojekyll']} />
-
-Files in the **public** directory are copied to the **dist** directory when you build your web app.
-
-</Step>
-
-<Step label="5">
-
 Modify the `scripts` in the **package.json** file by adding `predeploy` and `deploy` scripts. Each has its own value:
 
 {/* prettier-ignore */}
 ```json package.json
 "scripts": {
  /* @hide ... */ /* @end */
-  "deploy": "gh-pages -t -d dist",
+  "deploy": "gh-pages --nojekyll -d dist",
   "predeploy": "expo export -p web"
 }
 ```
 
-Running `gh-pages` with the `-t` flag is required. This flag ensures that the files starting with `.` are also included.
+Because Expo uses underscores in generated files, you need to disable jekyll with the `--nojekyll` flag.
```

Comment:
```suggestion
Since Expo uses underscores in generated files, you need to disable Jekyll with the `--nojekyll` flag.
```

Branding name and also let's not use the word "since" instead of because to start the sentence.